static/js/components/CreateCommentForm.js

```diff
@@ -18,7 +18,7 @@ type CreateCommentFormProps = {
   initialValue: CommentForm,
   formKey: string,
   beginReply: (initialValue: Object, e: ?Object) => void,
-  onSubmit: (p: string, t: string, c: string, p: Post) => () => void,
+  onSubmit: (p: string, t: string, c: string, p: Post, e: Object) => Promise<*>,
```

Comment:
Can you use `e: Event` here and in the calling functions?

Reply:
sure